src/client/components/LectureTable.js

```diff
@@ -45,11 +46,11 @@ export default class LectureTable extends Component {
         this.getAvailableLectures = this.getAvailableLectures.bind(this)
         this.handleCellClick = this.handleCellClick.bind(this)
         this.handleFilterValueChange = this.handleFilterValueChange.bind(this)
-        this.getLectures = this.getLectures.bind(this)
     }
 
     componentDidMount() {
         /* This method is called after first render */
+        socket.on('new-lecture', this.getAvailableLectures)
```

Comment:
1) Setter først opp at socket lytter på 'new-lecture', slik at når det lagres en ny lecture så vil klienten hente oppdatert liste. (linje 53)
2) Deretter kalles funksjonen 'getAvalableLectures()' for første gang for å sette _lectureList_ listen i tabellen som vises. (linje 54)